packages/theme-patternfly-org/components/footer/footer.js

```diff
@@ -16,227 +16,223 @@ import redhatLogo from "./RHLogo.png";
 import "./footer.css";
 import logo from '../../layouts/logo.svg';
 
-export class Footer extends React.Component {
-  render() {
-    return (
-      <React.Fragment>
-        <PageSection className="ws-org-pfsite-l-footer">
-          <Grid>
+export const Footer = () => (
+  <React.Fragment>
+    <PageSection key="footer-1" className="ws-org-pfsite-l-footer">
+      <Grid>
+        <GridItem
+          sm={12}
+          md={6}
+          mdOffset={1}
+          className="pf-u-mb-lg pf-u-mb-0-on-sm"
+        >
+          <Grid className="pf-u-py-xl">
+            <GridItem sm={6} md={4} className="pf-u-ml-md pf-u-ml-0-on-md">
+              <p
+                className="pf-c-title ws-org-pfsite-footer-menu-list-title"
+              >
+                QUICKLINKS
+              </p>
+              <Nav aria-label="Quick Links" className="pf-m-white">
+                <NavList className="pf-c-nav__list-footer">
+                  <li className="pf-c-nav__item">
+                    <Link
+                      className="pf-c-nav__link"
+                      to={"/get-started/about"}
+                      aria-label="Get started with PatternFly 4"
+                    >
+                      Get started
+                    </Link>
+                  </li>
+                  <li className="pf-c-nav__item">
+                    <Link
+                      className="pf-c-nav__link"
+                      to={"/documentation/react/components/aboutmodal"}
```

Comment:
This link won't work. Any plan to setup forwarding for old links to new ones?

Reply:
Some other footer links are not correct as well, maybe go through all of them?

Reply:
Yep! After fullscreen pages I'll run the site through a dead link checker and we'll resolve all broken links (hopefully) in 1 PR.